administrator/language/de-DE/de-DE.plg_authentication_ldap.ini

```
@@ -11,10 +11,14 @@ PLG_LDAP_FIELD_BASEDN_DESC="Der Basis-DN (Basis-Domain-Name) des LDAP-Servers, z
 PLG_LDAP_FIELD_BASEDN_LABEL="Basis-DN"
 PLG_LDAP_FIELD_EMAIL_DESC="LDAP-Eigenschaft, die die E-Mail-Adresse des Benutzers enthält."
 PLG_LDAP_FIELD_EMAIL_LABEL="Map: E-Mail"
+PLG_LDAP_FIELD_IGNORE_REQCERT_TLS_DESC="Wenn diese Option aktiviert ist, wird das Serverzertifikat ignoriert, was nützlich ist, wenn z.B. Samba 4 mit einem selbstsignierten Zertifikat ausgeführt wird."
+PLG_LDAP_FIELD_IGNORE_REQCERT_TLS_LABEL="Zertifikat ignorieren"
 PLG_LDAP_FIELD_FULLNAME_DESC="LDAP-Eigenschaft, die den vollständigen Namen des Benutzers enthält."
 PLG_LDAP_FIELD_FULLNAME_LABEL="Attribut: Voller Name"
 PLG_LDAP_FIELD_HOST_DESC="Zum Beispiel, openldap.example.com"
 PLG_LDAP_FIELD_HOST_LABEL="LDAP-Host"
+PLG_LDAP_FIELD_LDAPDEBUG_DESC="Aktiviert Debugging mit festeingestelltem Level 7"
```

```suggestion
PLG_LDAP_FIELD_LDAPDEBUG_DESC="LDAP Fehlermeldungen der Ebene 7 werden ausgegeben"
```
Many thanks to @chmst for the Input on Glip

---

```suggestion
PLG_LDAP_FIELD_LDAPDEBUG_DESC="Aktiviert intensives Debugging (Level 7)"
```

Hab noch einen Vorschlag basierend auf @Hackwar 's Idee.

---

> Suggested change 
> PLG_LDAP_FIELD_LDAPDEBUG_DESC="Aktiviert Debugging mit festeingestelltem Level 7"
> PLG_LDAP_FIELD_LDAPDEBUG_DESC="Aktiviert intensives Debugging (Level 7)"
> Commit suggestion  Add suggestion to batch
> Hab noch einen Vorschlag basierend auf @Hackwar 's Idee.

intensives ist mir persönlich zu hart.

---

und nur?
```suggestion
PLG_LDAP_FIELD_LDAPDEBUG_DESC="Aktiviert Debugging (Level 7)"
```

---

das "mit festeingestelltem " finde ich was holprig im deutschen